Classes/Report/AccessFilterPluginInstalledStatus.php

```diff
@@ -63,7 +65,7 @@ class AccessFilterPluginInstalledStatus implements StatusProviderInterface
     public function getStatus()
     {
         $reports = array();
-        $solrConnections = GeneralUtility::makeInstance('ApacheSolrForTypo3\\Solr\\ConnectionManager')->getAllConnections();
+        $solrConnections = GeneralUtility::makeInstance(ConnectionManager::class)->getAllConnections();
```

Comment:
I would say both is ok, since we could change it when we work on the code anyway. Doing this in multiple pull requests is better for sure, but i think we could also be pracmatically and change it when we work in these parts.